resources/web/wwi/FloatingProtoParameterWindow.js

```diff
@@ -158,36 +200,47 @@ export default class FloatingProtoParameterWindow extends FloatingWindow {
     resetButton.style.pointerEvents = 'all';
   }
 
+  #createBackButton(parent) {
+    const buttonContainer = document.createElement('span');
+    buttonContainer.style.gridRow = '' + this.#rowNumber + ' / ' + this.#rowNumber;
+    buttonContainer.style.gridColumn = '1 / 5';
+    buttonContainer.style.justifySelf = 'center';
+    buttonContainer.style.paddingTop = '15px';
+
+    const backButton = document.createElement('button');
+    backButton.innerHTML = 'Back';
+    backButton.title = 'Return to the previous PROTO';
+    backButton.onclick = () => {
+      this.proto = this.#protoManager.proto;
+      this.populateProtoParameterWindow();
+    };
+    buttonContainer.appendChild(backButton);
+    parent.appendChild(buttonContainer);
+  }
+
   #createDownloadButton(parent) {
-    const downloadlButton = document.createElement('button');
-    downloadlButton.innerHTML = 'Download';
     const buttonContainer = document.createElement('span');
     buttonContainer.className = 'value-parameter';
     buttonContainer.style.gridRow = '' + this.#rowNumber + ' / ' + this.#rowNumber;
     buttonContainer.style.gridColumn = '4 / 4';
 
+    if (typeof this.exportName === 'undefined')
+      this.exportName = this.#protoManager.proto.name
+
     const input = document.createElement('input');
     input.type = 'text';
     input.title = 'New proto name';
-    input.value = 'My' + this.#protoManager.proto.name;
+    input.value = this.exportName;
+    input.onchange = (e) => this.exportName = e.target.value;
```

Comment:
I saw that you removed the `My` prefix.
I added it to avoid conflicts with the base proto and also to show that the user can modify the name